adobe_vipm/adobe/utils.py

```
@@ -93,6 +93,24 @@ def get_3yc_commitment_request(customer, *, is_recommitment=False):  # noqa: WPS
     return benefit_3yc.get(recommitment_or_commitment, {}) or {}
 
 
+def get_3yc_recommitment_request(customer):  # noqa: WPS114
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: WPS114

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)